tests/integration/cloud/clouds/test_vmware.py

```diff
@@ -88,7 +88,11 @@ def test_instance(self):
         profile_config = cloud_config(profile)
         disk_datastore = profile_config['vmware-test']['devices']['disk']['Hard disk 2']['datastore']
 
+        for key, value in profile_config.get('vmware-test', {}).items():
+            assert value, "Instance config incomplete; missing '{}'".format(key)
```

Comment:
Do you want to blow up on the first problem here, or would it be better to collect potential issues?

Reply:
The errors I was getting before led me in endless circles, so yes, I want to blow up sooner